tests/bin/kvm_get_att_quote.c

```diff
@@ -0,0 +1,26 @@
+#include "libc.h"
+#include <errno.h>
+
+/* This test will be run only for KVM. It is designed to request a
+ * Quote from get_attestation() and check that the first bytes of
+ * the returned Quote in buf match expected values. */
+
+int main(void) {
+    int* nonce = NULL;
+    // TODO Update this buffer size to match real Quote.
+    unsigned char buf[16 * 1024];
+    size_t technology;
+    int i;
+
+    ssize_t size = get_att(nonce, sizeof(nonce), buf, sizeof(buf), &technology);
+
+    if ((size < 0) || (technology != 0))
+        return -1;
+
+    for (i = 0; i < sizeof(buf); i++) {
+        if (buf[i] != 0xEA)
+	    return 1;
+    }
+
+    return 0;
+}
```

Comment:
Same comment above, `get_attestation()` shouldn't be implemented for KVM.

Reply:
yeah